client/src/containers/filters/types.d.ts

```diff
@@ -0,0 +1,12 @@
+export type FiltersType =
```

Comment:
Should we move all types to general types folder?

Reply:
we should reorganize them definitely